lib/jsi/schema/application/child_application/contains.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module JSI
+  module Schema::Application::ChildApplication::Contains
+    # @private
+    def internal_applicate_contains(idx, instance, &block)
+      if schema_content.key?('contains')
+        contains_schema = subschema(['contains'])
+
+        if contains_schema.instance_valid?(instance[idx])
```

Comment:
bug: should be `validate_instance` here. failed to add `contains` to schema_child_application_test.